File: ProjectTwo(Python)/models/request_job.py

class RequestJob:

    def __init__(self, req_id=0, user_id=0, position_app="", job_status=""):
        self.req_id = req_id
        self.user_id = user_id
        self.position_app = position_app
        self.job_status = job_status

    def __repr__(self):
        return str({
            'req_id': self.req_id,
            'user_id': self.user_id,
            'position_app': self.position_app,
            'job_status': self.job_status
        })

    def json(self):
        return str({
            'reqId': self.req_id,
            'userId': self.user_id,
            'positionApp': self.position_app,
            'jobStatus': self.job_status
        })

File: JavaBackend/src/main/java/models/RequestJob.java
package models;

public class RequestJob {

    private int reqID;
    private int userID;
    private String positionApp;
    private String jobStatus;

    public RequestJob(){}

    public RequestJob(int reqID, int userID, String positionApp, String jobStatus){
        this.reqID=reqID;
        this.userID=userID;
        this.positionApp=positionApp;
        this.jobStatus=jobStatus;
    }

    public int getReqID() {
        return reqID;
    }

    public void setReqID(int reqID) {
        this.reqID = reqID;
    }

    public int getUserID() {
        return userID;
    }

    public void setUserID(int userID) {
        this.userID = userID;
    }

    public String getPositionApp() {
        return positionApp;
    }

    public void setPositionApp(String positionApp) {
        this.positionApp = positionApp;
    }

    public String getJobStatus() {
        return jobStatus;
    }

    public void setJobStatus(String jobStatus) {
        this.jobStatus = jobStatus;
    }
}
